translations/trans_7_10_17.csv

```diff
@@ -0,0 +1,613 @@
+Key,EN,ES
```

Comment:
👍  removed csv file, fixed the help page heading, and documented two other strings that are in need of translation.